### PR TITLE
dragonball: metrics support for vmm action requests.

### PR DESCRIPTION
**Pull request by lisongqian**
This PR adds metrics support for VMM action requests.

Fixes: #7248

Signed-off-by: Songqian Li <mail@lisongqian.cn>

### CODE REVIEW

File: src/runtime-rs/crates/hypervisor/src/dragonball/vmm_instance.rs [hunk 1]
```diff
@@ -152,6 +154,7 @@ impl VmmInstance {
     }
 
     pub fn put_boot_source(&self, boot_source_cfg: BootSourceConfig) -> Result<()> {
+        METRICS.read().unwrap().request.configure_boot_source.inc();
```
Comment (ZizhengBian) [inline src/runtime-rs/crates/hypervisor/src/dragonball/vmm_instance.rs:157]: should we add metrics here or add it to vmm_action.rs?

Comment (lisongqian) [reply src/runtime-rs/crates/hypervisor/src/dragonball/vmm_instance.rs:157]: Maybe not as useful. I removed it.

Comment (ZizhengBian) [reply src/runtime-rs/crates/hypervisor/src/dragonball/vmm_instance.rs]: You misunderstood my words.
I mean you can move the code in your patch to dragonball's vmm_action.rs, don't put it here in runtime's code

Comment (lisongqian) [reply src/runtime-rs/crates/hypervisor/src/dragonball/vmm_instance.rs]: Okay, I moved the code to  vmm_action.rs except for the request using `handle_request_with_retry`.

**Pull request closed without merging**